javaparser-core/src/main/java/com/github/javaparser/ast/type/Type.java

```diff
@@ -146,4 +152,27 @@ public boolean replace(Node node, Node replacementNode) {
         }
         return super.replace(node, replacementNode);
     }
+
+    public List<AnnotationExpr> getAnnotationsAtLevel(int level) {
```

Comment:
Every new method needs Javadoc, even if not all of JavaParser is properly documented.  For example, what are the legal arguments?  (Why is -1 legal for `level`?)  What are the possible return values (it might return null).
Should the method throw an exception if level is greater than the number of array dimensions?